### DISCUSSION TITLE
能新增一个“查看依赖该模组的模组”的选项吗？

### DISCUSSION THREAD

**Discussion by xl1533** [Meloong-Git/PCL#607] [提问] [answered]
貌似现在是有模组会显示依赖的功能（虽然有的时候还是会缺少），但是我还想要一个能查看把该模组作为依赖的模组的选项。

**Comment by WForst-Breeze** [comment, marked as answer]
有些模组的衍生模组太多了（如匠魂）
![image](https://user-images.githubusercontent.com/110760354/208018675-fe5f148c-776b-4b62-9ef2-3e3777339607.png)
一方面能完整地获取到这些信息是个问题，另一方面关于这个功能的UI设计也是个问题（小小一个启动器塞这么多元素可能会显得臃肿），关于此功能，你或许可以去[ mcmod.cn ](mcmod.cn)去逛逛…… 仅个人意见，如果你迫切需要这个功能，请考虑提交 [**新功能提案**](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)
欢迎友善讨论
